### ISSUE TITLE
[Meds / Tech Debt] Fix flaky e2e tests regarding the alert messages that include the alert type

### ISSUE THREAD

**Issue by amycashbaugh** [department-of-veterans-affairs/va.gov-team#129099]
## Context
While creating a [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/41163) to fix accessibility header issues, I discovered that several e2e tests (unrelated the changes that were made) were failing due to a change in the alert component implementation. The component now includes the alert type (Warning, Error, Information, etc.) along with the alert message.

## Screenshots

<img width="577" height="311" alt="Image" src="https://github.com/user-attachments/assets/a8d7c8e2-67e0-4733-a824-beafca72db07" />

<img width="600" height="295" alt="Image" src="https://github.com/user-attachments/assets/2fae0e4b-797a-4529-88fe-7dca3f345a7f" />

## Possible Tasks
- [ ] Determine why the tests are failing
- [ ] Fix them

## Acceptance Criteria
- [ ] Tests are fixed
- [ ] Code Review
